### ISSUE TITLE
[Slint ver 1.0.1] CompilerConfiguration does not change the rendering style to "material"

### ISSUE THREAD

**Issue by volodg** [slint-ui/slint#2610]
Given code does not work anynore:

let config = slint_build::CompilerConfiguration::new().with_style("material".into());
slint_build::compile_with_config("ui/app.slint", config).unwrap();

OS: MacOS

**Comment by ogoffart** [comment]
Thanks for reporting a bug.

 Could you elaborate what is not working exactly? What is the error.
Thanks

**Comment by volodg** [comment]
No errors, style just does change to "material", and default one is displayed

**Comment by volodg** [comment]
This app was looking like that with version 1.0.0
![image](https://user-images.githubusercontent.com/32120/233592082-df7dbf3c-393e-4d60-9616-99c7aa7af792.png)

And now:
<img width="617" alt="Screenshot 2023-04-21 at 10 50 10" src="https://user-images.githubusercontent.com/32120/233592128-513dd7b5-72a9-4ca2-9659-a07b2e070e55.png">

The magic is, downgrading version does not help ((

Source code

When I run this app https://github.com/biotech-ng/exchange/blob/main/slint/src/main.rs

**Comment by ogoffart** [comment]
Thanks for the screenshots.
Both are showing the material theme.  One is showing the light theme, and the other is showing the dark theme.

Something happened that made it detect a different dark mode.

You can force the light mode by using  "material-light":

```
let config = slint_build::CompilerConfiguration::new().with_style("material-light".into());
```

But is your desktop in dark mode or not?  
Maybe somehow the style detection was either fixed, or broken.  (Possibly because dependency upgrades)

**Comment by volodg** [comment]
I have a dark style in MacOS, right, so seems it was not broken but fixed in 1.0.1, now with the style: "material-light" it works as expected, thanks for your advise. Closing an issue
